packages/orchestration/src/exos/orchestrator.js

```diff
@@ -126,6 +126,7 @@ const prepareOrchestratorKit = (
             return asVow(() => chainByName.get(name));
           }
           if (name === 'agoric') {
+            // @ts-expect-error types of property 'chainId' are incompatible (string vs a chainId const)
```

Comment:
Wasn't sure on this one. Is it addressable?

Reply:
This is because the runtime supports more than KnownChains. A bunch of the places that were changed from `string` to `KnownChainName` I think should stay `string` and things that really expect a KnownChainName can look in knownChains to figure out whether it is or isn't.

The alternative is that we make developers do some build step in their local env to define KnownChainName that's different than the one in agoric-sdk. Let's punt on that because we can expect agoric-sdk to know all the chains pretty quickly. We can push NPM releases pretty fast. And in a pinch they could patch the package.

Though considering that… maybe we should just have a type `ChainName` and it default to all the known ones but someone can override it with another they know.

Let's discuss